beprof/unittests.py

```diff
@@ -0,0 +1,115 @@
+from beprof.curve import Curve, DataSerie
+from beprof.profile import Profile, LateralProfile
+import numpy as np
+import unittest
+import copy
+
+cur = Curve([[0, 0], [1, 1], [2, 2], [3, 2], [4, 3], [5, 4], [6, 4], [7, 3], [8, 2], [9, 1], [10, 0]],
+            arg1 = 'first',
+            arg2 = 'second'
+            )
+pro = Profile([[0, 0], [1, 1], [2, 2], [3, 2], [4, 3], [5, 4], [6, 4], [7, 3], [8, 2], [9, 1], [10, 0]],
+            profile1 = 'first_prof',
+            profile2 = 'second_prof'
+            )
+lpr = LateralProfile([[0, 0], [1, 1], [2, 2], [3, 2], [4, 3], [5, 4], [6, 4], [7, 3], [8, 2], [9, 1], [10, 0]],
+            lprof1 = 'first_lp',
+            lprof2 = 'second_lp'
+            )
+
+
+class TestCurveMethods(unittest.TestCase):
+
+    def test_metadata_1(self):
```

Comment:
I think this 3 checks in test_metadata_1 should go to 3 different functions. If the test fail it would be easier to find out why.